frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -73,6 +79,25 @@ function CriticalQuestions(props) {
         setTravelDates({ ...travelDates, [travelDatesIndex]: date });
     };
 
+const handleSubmit = (event) => {
```

Comment:
Indentation of `handleSubmit` is a little bit confusing here. It should be aligned at the same level of other definitions. It may be corrected when we start using the linter, though, so it's up to you